pkg/swagger/docs.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by swaggo/swag. DO NOT EDIT
+// Package swagger Code generated by swaggo/swag. DO NOT EDIT
```

Comment:
I know we just discussed the benefits and potential of generating the swagger definitions manually instead of using swaggo, but looking at this huge file it scares me to manage it manually. Specially that it includes specifications of things like `models.Job` and `models.Executions`. 

Is there a middle ground where the models definitions are auto-generated, but the API specs are manually generated and documented? We don't need to think about this now, but something for later when we decide to improve our clients generation